torch/utils/_sympy/interp.py

```diff
@@ -22,6 +22,9 @@
 @functools.lru_cache(None)
 def handlers():
     from torch.fx.experimental.symbolic_shapes import FloorDiv, Pow, TrueDiv
+    from torch._inductor.ir import CleanDiv
+
+    # TODO: CeilDiv, ModularIndexing
```

Comment:
CleanDiv is the one that showed up when traversing sympy expressions, CeilDiv/ModularIndexing didn't show up in testing.